zeebe/protocol/src/main/java/io/camunda/zeebe/protocol/record/intent/AuthorizationIntent.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.protocol.record.intent;
+
+public enum AuthorizationIntent implements Intent {
+  CREATE(0),
+  CREATED(1),
+  UPDATE(2),
+  UPDATING(3),
+  UPDATED(4),
+  DELETE(5),
+  DELETING(6),
+  DELETED(7);
```

Comment:
❌ Let's remove these for now. We aren't using them. I think we don't need `UPDATING` and `DELETING` anyways but let's think about that when we implements those features.